p2p/peer.go

```diff
@@ -45,6 +46,15 @@ func (pc peerConn) ID() ID {
 	return PubKeyToID(pc.conn.(*tmconn.SecretConnection).RemotePubKey())
 }
 
+// Return the IP from the connection RemoteAddr
+func (pc peerConn) RemoteIP() string {
```

Comment:
If we truly want to return an ip address I think we should do a lookup in case `RemoteAddr()` is a dns name.